string[] name
---
uint32[] found
float64[] position
float64[] velocity
float64[] effort
float64[] goal_pos
float64[] error
float64[] temp

